app/controllers/rooms_controller.rb

```diff
@@ -4,6 +4,7 @@ def new
 
   def create
     room = Room.new(room_params)
+    room.choices.build(title: params[:room][:choice])
```

Comment:
As it stands now, a room must have at least one choice to be valid, it will not save without that relationship established. We could remove the validation on room, but I don't think we should ever have a room that no choice